### PR TITLE
[export] modify export code owners

### PR DESCRIPTION
**Pull request by ydwu4**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #130894

### CODE REVIEW

**Comment by zhxchen17** [comment]
@ydwu4 can you also add angela here lol

File: CODEOWNERS [hunk 1]
```diff
@@ -143,8 +143,8 @@ caffe2/utils/hip @jeffdaily @jithunnair-amd
 /third_party/xpu.txt @EikanWang @gujinghui
 
 # torch.export
-/torch/export/ @avikchaudhuri @gmagogsfm @tugsbayasgalan @zhxchen17
-/torch/_export/ @avikchaudhuri @gmagogsfm @tugsbayasgalan @zhxchen17
+/torch/export/ @avikchaudhuri @gmagogsfm @tugsbayasgalan @zhxchen17 @ydwu4
+/torch/_export/ @avikchaudhuri @gmagogsfm @tugsbayasgalan @zhxchen17 @ydwu4
```
Comment (zhxchen17) [inline CODEOWNERS:147]: sorry maybe also remove gmagogsfm

Comment (ydwu4) [reply CODEOWNERS]: sure!

**Review by zhxchen17 — APPROVED** [review]

**Pull request closed without merging**